tools/pipelines/templates/build-npm-package.yml

```diff
@@ -1,598 +1,540 @@
+              - '${{ if eq(parameters.taskPublishBundleSizeArtifacts, false) }}':
+                  - template: >-
+                      /tools/pipelines/templates/include-set-package-version.yml@self
+                    parameters:
+                      buildDirectory: '${{ parameters.buildDirectory }}'
+                      buildNumberInPatch: '${{ parameters.buildNumberInPatch }}'
+                      buildToolsVersionToInstall: '${{ parameters.buildToolsVersionToInstall }}'
+                      tagName: '${{ parameters.tagName }}'
+                      interdependencyRange: '${{ parameters.interdependencyRange }}'
+                      packageTypesOverride: '${{ parameters.packageTypesOverride }}'
+              - template: /tools/pipelines/templates/include-build-lint.yml@self
+                parameters:
+                  taskBuild: '${{ parameters.taskBuild }}'
+                  taskLint: '${{ parameters.taskLint }}'
+                  taskLintName: '${{ parameters.taskLintName }}'
+                  buildDirectory: '${{ parameters.buildDirectory }}'
+              - '${{ if ne(convertToJson(parameters.taskTest), ''[]'') }}':
+                  - script: |
+                      echo "##vso[task.setvariable variable=startTest]true"
+                    displayName: Start Test
+                  - '${{ each taskTestStep in parameters.taskTest }}':
+                      - template: /tools/pipelines/templates/include-test-task.yml@self
+                        parameters:
+                          taskTestStep: '${{ taskTestStep }}'
+                          buildDirectory: '${{ parameters.buildDirectory }}'
+                  - '${{ if eq(variables[''testCoverage''], true) }}':
+                      - task: Bash@3
+                        displayName: Check for nyc/report directory
+                        inputs:
+                          targetType: inline
+                          workingDirectory: '${{ parameters.buildDirectory }}'
+                          script: >
+                            test -d nyc/report && echo '##vso[task.setvariable
+                            variable=ReportDirExists;]true' || echo 'No
+                            nyc/report directory'
+                        condition: >-
+                          and(succeededOrFailed(), eq(variables['startTest'],
+                          'true'))
+                      - task: Bash@3
+                        displayName: Patch Coverage Results
+                        inputs:
+                          targetType: inline
+                          workingDirectory: '${{ parameters.buildDirectory }}/nyc/report'
+                          script: >
+                            sed -e 's/\(filename=\".*[\\/]external
+                            .*\)"\(.*\)""/\1\&quot;\2\&quot;"/'
+                            cobertura-coverage.xml >
+                            cobertura-coverage-patched.xml
```

Comment:
Same as above, breaking a sed command like this is not good for readability. A few more in the lines beneath this one.